animations/test/misc/expand_card_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:animations/src/misc/expand_card.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+Widget createExpandCardScreen() => MaterialApp(
+      home: ExpandCardDemo(),
+    );
+
+void main() {
+  testWidgets('size of ExpandCard changes on tap', (tester) async {
+    await tester.pumpWidget(createExpandCardScreen());
+
+    Size size = tester.getSize(find.byType(AnimatedContainer));
+
+    // size of Expand Card is 128 initially
```

Comment:
Throughout the repo, comments should start with a capital letter and end with a period.